tools/integration_tests/util/client/gcs_helper.go

```diff
@@ -78,6 +78,21 @@ func ValidateObjectContentsFromGCS(ctx context.Context, storageClient *storage.C
 	}
 }
 
+func ValidateObjectChunkFromGCS(ctx context.Context, storageClient *storage.Client,
+	testDirName string, fileName string, offset, size int64, expectedContent string,
+	t *testing.T) {
+	gotContent, err := ReadChunkFromGCS(ctx, storageClient,
```

Comment:
This should be expectedContent and in above should be got content?

Reply:
Discussed offline, the "expected" and "got" depends on the context of test and what we are considering as our source of truth. We are not changing the variable names here to keep it consistent with other similar methods in gcs_helper.go